### PR TITLE
Implement profiles activations with precompiled script plugin (Renan)

### PR DESCRIPTION
**Pull request by renanfranca**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -191,6 +184,27 @@ dependencies {
   // jhipster-needle-gradle-test-dependencies
 }
 
+val profile = project.findProperty("profile") as String? ?: "default"
```
Comment (murdos) [inline build.gradle.kts]: You seems to be missing the poing that multiple build profiles can be active by default, and they should all be activated.

Comment (murdos) [reply build.gradle.kts]: This PR contains a 2nd commit that improves things: https://github.com/renanfranca/gradleapp-profiles-sample/pull/2/files

Here's how it works:
- `./gradlew build -x test --rerun-tasks -Pprofiles=local && grep "active:" build/resources/main/config/application.yml` -> `active: "local"`
- `./gradlew build -x test --rerun-tasks -Pprofiles=dev && grep "active:" build/resources/main/config/application.yml` -> `active: "dev"`
- `./gradlew build -x test --rerun-tasks && grep "active:" build/resources/main/config/application.yml` -> `active: "local"`

Comment (renanfranca) [reply build.gradle.kts]: > You seems to be missing the poing that multiple build profiles can be active by default, and they should all be activated.

Sorry, I completely missed that rule. I wasn't considering it. Thanks

Comment (renanfranca) [reply build.gradle.kts]: Thanks @murdos ! Can I start the implementation? WDYT?

Comment (murdos) [reply build.gradle.kts]: Yes, let's try that way! 🙂

Comment (renanfranca) [reply build.gradle.kts]: > Yes, let's try that way! 🙂

Thank you very much, let's go! I will start tomorrow morning 👍

**Pull request merged**